srcpkgs/supersonic-desktop/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'supersonic-desktop'
+pkgname=supersonic-desktop
+version=0.8.2
+revision=1
+build_style=go
+go_import_path="github.com/dweymouth/supersonic"
+hostmakedepends="pkg-config gcc"
+makedepends="libglvnd-devel mpv-devel libX11-devel libappindicator-devel libsecret-devel libXxf86vm-devel"
+depends="libsecret libappindicator"
+short_desc="Full-featured, mpv-based desktop client written in go for Subsonic and Jellyfin servers"
```

Comment:
```suggestion
short_desc="Full-featured, mpv-based music player written in go for Subsonic and Jellyfin servers"
```

I feel like mentioning `music`  somewhere in the description is of value, since this is a music player after all

Reply:
Fixed:
- Changed desc to proposed by @Johnnynator 
- replaced appindicator (just removed from repo) with libayatana-appindicator (works good, compiled tested)
- Replaced GPL-3.0 with GPL-3.0-or-later